sdk/core/azure-core/inc/azure.hpp

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I don't think we want this behavior.

This says put the `azure.hpp` file at `\inc\` on a unix system, i.e.:
`\inc\azure.hpp`

I expect that I would find all Azure SDK headers under the azure folder, i.e.:
`\inc\azure\azure.hpp`

Reply:
updated.  Since we have core.hpp in `/inc/azure/core.hpp` , I thought we wanted the same for azure.hpp.  I don't have a personal preference here